metaci/build/models.py

```diff
@@ -428,9 +428,7 @@ def run(self):
 
         if self.plan.role == "release":
             try:
-                send_release_webhook(
-                    project_config, self.release, self.org.configuration_item
-                )
+                send_release_webhook(self.release, self.org.configuration_item)
```

Comment:
Currently we use this to create change cases automatically after running an Upload Release plan.

But since cases need to be approved before we can start their implementation steps, we are switching to creating them ahead of time using the Create Change Case button.

I think that means we need to remove lines 429-438 here so that we don't end up creating a duplicate case after the plan runs. Does that make sense?